### PR TITLE
Docs/psql9.0 docs merge b

### PR DESCRIPTION
**Pull request by litzell**
Edits to GPDB docs from sgml files updated by the 2nd postgresql 9.0 merge
#4645

Staged docs are here: http://docs-litzell-psql90-merge-b.cfapps.io/600/ref_guide/sql_commands/sql_ref.html

## Here are some reminders before you submit the pull request
- [ ] Add tests for the change
- [ ] Document changes
- [ ] Communicate in the mailing list if needed
- [ ] Pass `make installcheck`

### CODE REVIEW

File: gpdb-doc/dita/ref_guide/sql_commands/SELECT.xml [hunk 1]
```diff
@@ -910,14 +957,18 @@ SELECT distance, employee_name FROM employee_recursive;</codeblock>
...
+        </p><sectiondiv id="section20"><b>Omitted FROM Clauses</b><p>Greenplum Database allows one to omit the
+            <codeph>FROM</codeph> clause. It has a straightforward use to compute the results of
+          simple expressions. For example:</p><codeblock>SELECT 2+2;</codeblock><p>Some other SQL
+          databases cannot do this except by introducing a dummy one-row table from which to do the
+            <codeph>SELECT</codeph>. </p><p>Note that if a <codeph>FROM</codeph> clause is not
+          specified, the query cannot reference any database tables. For example, the following
+          query is
+          invalid:<codeblock>SELECT distributors.* WHERE distributors.name = 'Westward';</codeblock>In
+          earlier releases, setting a server configuration parameter,
+            <varname>add_missing_from</varname>, to true allowed Greenplum Database to add an
```
Comment (lisakowen) [inline gpdb-doc/dita/ref_guide/sql_commands/SELECT.xml]: not related to this PR:  i am not familiar with the "add_missing_from" GUC.  is it just no longer used here (but used elsewhere), or was it removed from the code and should also removed from the docs?  (i don't see it in the code anywhere...)

File: gpdb-doc/dita/ref_guide/sql_commands/ALTER_TABLESPACE.xml [hunk 2]
```diff
@@ -3,10 +3,31 @@
...
+          <pd>A tablespace parameter to be set or reset. Currently, the only available parameters
+                        are <varname>seq_page_cost</varname> and
+                        <varname>random_page_cost</varname>. Setting either value for a particular
+                        tablespace will override the planner's usual estimate of the cost of reading
+                        pages from tables in that tablespace, as established by the configuration
+                        parameters of the same name (see <xref
+                                href="../config_params/guc-list.xml#seq_page_cost" format="dita" scope="local"
+                                >seq-page-cost</xref>, <xref
+                                href="../config_params/guc-list.xml#random_page_cost"
+                                format="dita" scope="local">random-page-cost</xref>).This may be useful if one
```
Comment (dyozie) [inline gpdb-doc/dita/ref_guide/sql_commands/ALTER_TABLESPACE.xml]: missing space before "This"

File: gpdb-doc/dita/ref_guide/sql_commands/CREATE_INDEX.xml [hunk 3]
```diff
@@ -298,16 +303,24 @@
...
+                        <codeblock>CREATE INDEX ON films ((lower(title)));</codeblock>
+                        <p>(In this example we have chosen to omit the index name, so the system
+                                will choose a name, typically <codeph>films_lower_idx</codeph>.)
+                        </p>
                         <p>To create an index with non-default fill factor:</p>
                         <codeblock>CREATE UNIQUE INDEX title_idx ON films (title) WITH 
 (fillfactor = 70);</codeblock>
                         <p>To create a GIN index with fast updates disabled:</p>
-                        <codeblock>CREATE INDEX gin_idx ON documents_table (locations) WITH (fastupdate = off);</codeblock>
+                        <codeblock>CREATE INDEX gin_idx ON documents_table USING gin(locations) WITH (fastupdate = off);</codeblock>
```
Comment (dyozie) [inline gpdb-doc/dita/ref_guide/sql_commands/CREATE_INDEX.xml]: missing space after "gin"

File: gpdb-doc/dita/ref_guide/sql_commands/CREATE_LANGUAGE.xml [hunk 4]
```diff
@@ -23,9 +25,24 @@ CREATE [TRUSTED] [PROCEDURAL] LANGUAGE <varname>name</varname>
...
                 only trusted languages to be registered by database owners. The creator of a
                 language becomes its owner and can later drop it, rename it, or assign ownership to
-                a new user.</p>
-            <p><codeph>CREATE LANGUAGE</codeph> effectively associates the language name with a call
-                handler that is responsible for executing functions written in that language. For a
+                a new owner.</p>
+            <p><codeph>CREATE OR REPLACE LANGUAGE</codeph> will either create a
+                new language, or replace an existing definition. If the language
+                already exists, its parameters are updated according to the values
+                specified or taken from <varname>pg_pltemplate</varname>,
```
[the remark this answers is not in the document]
Comment (dyozie) [reply gpdb-doc/dita/ref_guide/sql_commands/CREATE_LANGUAGE.xml]: I'd opt not to in this case.  Getting into the details of the catalog definition seems advanced, so I'm fine with the reader having to search for this if they have the inclination.

File: gpdb-doc/dita/ref_guide/sql_commands/ALTER_ROLE.xml [hunk 5]
```diff
@@ -41,33 +42,35 @@ ALTER ROLE <varname>name</varname> [ [WITH] <varname>option</varname> [ ... ] ]<
...
-            user-settable configuration parameters.</p></li>
+        <li id="av136604"><b>SET | RESET</b> — Changes a role's session default for a specified
+          configuration parameter, either for all databases or, when the <codeph>IN
+            DATABASE</codeph> clause is specified, only for sessions in the named database. If
+            <codeph>ALL</codeph> is specified instead of a role name, this changes the setting for
+          all roles. Using <codeph>ALL</codeph> with <codeph>IN DATABASE</codeph> is effectively the
+          same as using the command <codeph>ALTER DATABASE...SET...</codeph>. <p>Whenever the role
+            subsequently starts a new session, the specified value becomes the session default,
+            overriding whatever setting is present in the server configuration file
+              (<codeph>postgresql.conf</codeph>) or has been received from the postgres command
```
[the remark this answers is not in the document]
Comment (litzell) [reply gpdb-doc/dita/ref_guide/sql_commands/ALTER_ROLE.xml]: done

File: gpdb-doc/dita/ref_guide/sql_commands/ALTER_ROLE.xml [hunk 6]
```diff
@@ -41,33 +42,35 @@ ALTER ROLE <varname>name</varname> [ [WITH] <varname>option</varname> [ ... ] ]<
...
+            line. This only happens at login time; executing <xref href="SET_ROLE.xml#topic1">SET
+              ROLE</xref> or <xref href="SET_SESSION_AUTHORIZATION.xml#topic1">SET SESSION
+                AUTHORIZATION</xref> does not cause new configuration values to be set. </p>   
+          <p>Settings set for all databases are overridden by database-specific settings
```
[the remark this answers is not in the document]
Comment (litzell) [reply gpdb-doc/dita/ref_guide/sql_commands/ALTER_ROLE.xml]: I think clarity is a good reason to diverge from postgres.

File: gpdb-doc/dita/ref_guide/sql_commands/ALTER_ROLE.xml [hunk 7]
```diff
@@ -41,33 +42,35 @@ ALTER ROLE <varname>name</varname> [ [WITH] <varname>option</varname> [ ... ] ]<
...
+          attached to a role.</p>
+          <p>For
+            a role without <codeph>LOGIN</codeph> privilege, session defaults have no effect.
+            Ordinary roles can change their own session defaults. Superusers can change anyone's
+            session defaults. Roles having <codeph>CREATEROLE</codeph> privilege can change defaults
+            for non-superuser roles. Ordinary roles can only set defaults for themselves. Certain 
+            configuration variables cannot be set this way, or can only be set if a superuser 
+            issues the command. See the <i>Greenplum Database Administrator Guide</i> for
```
[the remark this answers is not in the document]
Comment (litzell) [reply gpdb-doc/dita/ref_guide/sql_commands/ALTER_ROLE.xml]: I changed it to Reference guide. It's not in the admin guide, for sure!

File: gpdb-doc/dita/ref_guide/sql_commands/SELECT.xml (hunk 1 above)
Comment (litzell) [reply gpdb-doc/dita/ref_guide/sql_commands/SELECT.xml]: There is a GPDB story to remove that GUC from docs. It got pulled in 9.0.

File: gpdb-doc/dita/ref_guide/sql_commands/CREATE_TABLE.xml [hunk 8]
```diff
@@ -312,6 +338,24 @@
           <pd>Specifies if the column is or is not allowed to contain null values.
               <codeph>NULL</codeph> is the default.</pd>
         </plentry>
+        <plentry>
+          <pt>CHECK( <varname>expression</varname> )</pt>
+          <pd>The <varname>CHECK</varname> clause specifies an expression producing a
+            Boolean result which new or updated rows must satisfy for an
```
[the remark this answers is not in the document]
Comment (litzell) [reply gpdb-doc/dita/ref_guide/sql_commands/CREATE_TABLE.xml]: TRUE/UNKNOWN/FALSE yes, Boolean I don't think so.

**Pull request merged**